dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/WorkerGroupServiceImpl.java

```diff
@@ -400,4 +400,22 @@ public Map<String, Object> getWorkerAddressList() {
         return result;
     }
 
+    /**
+     * get all worker group names
+     *
+     * @return worker group name list
+     */
+    @Override
+    public List<String> getAllWorkerGroupNames() {
+        List<WorkerGroup> workerGroups = getWorkerGroups(false, null);
+        List<String> availableWorkerGroupList = workerGroups.stream()
+            .map(WorkerGroup::getName)
+            .collect(Collectors.toList());
+        if (!availableWorkerGroupList.contains(Constants.DEFAULT_WORKER_GROUP)) {
```

Comment:
availableWorkerGroupList maybe null.

Reply:
> availableWorkerGroupList maybe null.

It will not be null, because mybatis mapper will return a empty list, not null. Mybatis just will return null if return type is a DTO. thx